roles/st2/tasks/user.yml

```diff
@@ -10,7 +10,6 @@
     ssh_key_file: "{{ st2_ssh_key_file }}"
     state: present
   register: _user
-  tags: [st2, user]
```

Comment:
No tags at all? I can see not using `st2`, but the other tasks in this role have a tag; was just curious why this is being removed entirely